src/views/migrationpolicies/components/MigrationPolicyConfigurations/MigrationPolicyConfigurations.tsx

```diff
@@ -41,11 +45,25 @@ const MigrationPolicyConfigurations: React.FC<MigrationPolicyConfigurationsProps
         state={state}
       />
       {hasConfigSelected && (
-        <Form isHorizontal>
+        <Form className="migration-configuration-dropdown" isHorizontal>
           {Object.entries(options).map(
-            ([key, { component: Component, label }]) =>
+            ([key, { component: Component, label, labelHelp = null }]) =>
               key in state && (
                 <FormGroup
+                  labelHelp={
+                    labelHelp && (
+                      <HelpTextIcon
+                        bodyContent={
+                          <Trans ns="plugin__kubevirt-plugin" t={t}>
```

Comment:
Trans is not needed since you already translate it in the labelHelp.helpText directly